### DISCUSSION TITLE
single-node multi-drive for Veeam backup storage

### DISCUSSION THREAD

**Discussion by ste7n** [minio/minio#16873] [Q&A] [answered]
Hi All,

we would like to use a MinIO node configured as single-node multi-drive to host Veeam backups. 
The backup file volume is from 200 to 300 TB and the idea is to use a single Sipermicro StorageServer with multiple disks as a S3 storage.

Considering the uptime is not a killer factor, this setup is safe for the data?
What is the performance I can expect from the system?
_MinIO supports unlimited number of disks but each erasure set can be upto 16 disk_s rule can be an issue in a 60 disk storage server?

Thank you.
regards.
Stefano

**Comment by klauspost** [comment, marked as answer]
We only provide setup guidance as part of our [subscriptions](https://min.io/pricing). You can write `sales@min.io` and we can arrange a setup analysis.

> MinIO supports unlimited number of disks but each erasure set can be upto 16 disk_s rule can be an issue in a 60 disk storage server?

MinIO will select a set size of 12 in that case.

**Comment by stegitto** [comment]
Hi klauspost, 

thank you for the reply.
May I ask you if a single-node multi-drive is a supported scenario for the professional services?

Regards.
Stefano

**Reply by harshavardhana** [reply]
We do not do any professional services 
, It's only support for your deployments that comes with architecture reviews.
